Framework/DataHandling/inc/MantidDataHandling/SaveCanSAS1D.h

```diff
@@ -83,7 +83,9 @@ class MANTID_DATAHANDLING_DLL SaveCanSAS1D : public API::Algorithm {
 public:
   virtual ~SaveCanSAS1D() = default;
   const std::string name() const override { return "SaveCanSAS1D"; }
-  const std::string summary() const override { return "Save a MatrixWorkspace to a file in the canSAS 1-D format"; }
+  const std::string summary() const override {
+    return "Save a MatrixWorkspace to a file in the CanSAS1D XML format for 1D data.";
```

Comment:
In all of these summary updates, Steve's requested changes had brackets around the type of data that the algorithm is intended for, i.e. `Save a MatrixWorkspace to a file in the CanSAS1D XML format (for 1D data).`. These things can be a bit subjective, but I can see how including the brackets may make it a bit clearer that we're referring to the type of data that this algorithm is suitable for. I could see how excluding the brackets could potentially leave open an interpretation that the algorithm is outputting a particular CanSAS1D format designed for 1D data specifically.